src/js/filter.ts

```diff
@@ -91,23 +91,38 @@ const changeSelectedMarkers = (
   searchElement: Choices,
   sliders: PopulationSliders
 ) => {
+  let updatedCounter = 0;
```

Comment:
Nit: maybe something like `selectedCount` or `cityCount`? The part about it being `updated` is less relevant imo.